pkg-manager/core/src/install/index.ts

```diff
@@ -909,6 +931,7 @@ export interface UpdatedProject {
 }
 
 interface InstallFunctionResult {
+  newDefaultCatalogs?: Record<string, ResolvedCatalogEntry>
```

Comment:
This is declared a lot. So, maybe worth adding a type.

Reply:
@KSXGitHub Could we use the `Catalogs` type from `@pnpm/catalogs.types`?